internal/rpc/server_test.go

```diff
@@ -95,11 +95,7 @@ func runGrpcWithProxyTests(assert *assert.Assertions, s grpcServerWithProxy, con
 	httpResp, err := httpClient.Do(httpReq)
 	assert.Nil(err)
 	assert.NotNil(httpResp)
-	defer func() {
-		if httpResp != nil {
-			httpResp.Body.Close()
-		}
-	}()
+	defer httpResp.Body.Close()
```

Comment:
assert.Nil does not stop the test case.  It will have an error, then panic at this defer.
Is there a better way to do this that makes golangci happy?

Reply:
this [golangci check](https://github.com/timakin/bodyclose) only accept this body close pattern. I think we have to disable it when running the checks in our test files.